src/containers/PipelineResourcesDropdown/PipelineResourcesDropdown.js

```diff
@@ -37,11 +37,14 @@ class PipelineResourcesDropdown extends React.Component {
   }
 
   render() {
-    const { namespace, ...rest } = this.props;
-    const emptyText =
-      namespace === ALL_NAMESPACES
-        ? `No Pipeline Resources found`
-        : `No Pipeline Resources found in the '${namespace}' namespace`;
+    const { namespace, type, ...rest } = this.props;
+    let emptyText = `No Pipeline Resources found`;
```

Comment:
We'll need multiple strings here as concatenation won't work for translation

Reply:
Bummer, so I'll have to go through all the combinations of those two booleans?

Reply:
We could simplify the strings, omit the type and rely on the label / other context to make it clear to the user. Otherwise yes, we'd need one string per combo.